### PR TITLE
Rename, PyPi package, use Apache license as per Actian standard for open source projects

### PR DESCRIPTION
**Pull request by clach04**
Addresses:
  * https://github.com/ActianCorp/sqlalchemy-ingres/issues/10
  * https://github.com/ActianCorp/sqlalchemy-ingres/issues/11
  * https://github.com/ActianCorp/sqlalchemy-ingres/issues/12
  * https://github.com/ActianCorp/sqlalchemy-ingres/issues/16

### CODE REVIEW

**Comment by clach04** [comment]
@hab6 I opened https://github.com/ActianCorp/sqlalchemy-ingres/issues/26 (to be tackled after this is merged).

**Review by hab6 — APPROVED** [review]
LGTM

File: lib/sqlalchemy_ingres/_version.py
Comment (hab6) [inline lib/sqlalchemy_ingres/_version.py:1]: I just noticed this when comparing versions of the existing `ingres_sa_dialect` package versus the `sqlalchemy-ingres` package. Referencing the [Semantic Versioning doc](https://semver.org/), it looks like the `ingres_sa_dialect` package uses only **major.minor** but does not specify the **patch** level whereas with this PR, the `sqlalchemy-ingres` package uses **major.minor.patch**.

     C:\Users\mhabiger\Data\test\sqlalchemy\jan31
     (.venv) λ pip list | grep ingres
     ingres_sa_dialect 0.4
     sqlalchemy-ingres 0.0.4.dev1 C:\Users\mhabiger\Data\test\sqlalchemy\jan31\.venv\src\sqlalchemy-ingres

Comment (clach04) [reply lib/sqlalchemy_ingres/_version.py:1]: @hab6 Any concerns with that? This matches most other Python packages, for example sqlalchemy - see https://pypi.org/project/SQLAlchemy/2.0.25/

If all good, please merge. If concerns, let's get those figured out (e.g. documenting old name/version).

Comment (hab6) [reply lib/sqlalchemy_ingres/_version.py:1]: Fine with me. Just wanted to make sure I understood the reasoning. I agree with the 3 part version. Thanks!

**Pull request merged**